README.md

```diff
@@ -94,7 +94,7 @@ On a unix system, the build directory should now look something like this:
 Next, you can run the tests to check the build.
 
 ```bash
-$ cmake -E chdir "build" ctest ../
+$ cmake -E chdir "build" ctest --build-config Release ../
```

Comment:
Why do you keep this trailing `../`? My tests and specs suggests this has no meaning.